src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -22,6 +22,15 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_LinkSemaphore>$(IntermediateOutputPath)Link.semaphore</_LinkSemaphore>
   </PropertyGroup>
 
+  <!-- We disable startup hooks for trimmed apps here so that the feature
```

Comment:
Should this go into a props file so that it can actually be overriden from a project?

Reply:
It can't go into a .props file because then the project wouldn't get a chance to set `PublishTrimmed` before this check happens.

So instead, the check below should be 

`<PropertyGroup Condition="'$(StartupHookSupport)' == '' AND '$(PublishTrimmed)' == 'true'>`

which will allow for both to be set in the project file.